ArcGISRuntimeSDKQt_QMLSamples/EditData/ContingentValues/ContingentValues.qml

```diff
@@ -0,0 +1,374 @@
+                    newFeature.attributes.replaceAttribute("Status", statusValues[statusComboBox.currentValue]);
+
+                    // Append the valid contingent coded values to the subsequent ComboBox
+                    protectionComboBox.model = [""].concat(getContingentValues("Protection", "ProtectionFieldGroup"));
+                }
+            }
+
+            Text {
+                text: "Protection"
+                font {
+                    bold: true
+                    pointSize: 11
+                }
+            }
+
+            ComboBox {
+                id: protectionComboBox
+                model: [""];
+
+                enabled: statusComboBox.currentText !== ""
+
+                onCurrentValueChanged: {
+                    if (protectionComboBox.currentValue === "")
+                        return;
+
+                    // Update the feature's attribute map with the selection
+                    newFeature.attributes.replaceAttribute("Protection", protectionValues[protectionComboBox.currentValue]);
+
+                    // Get the valid contingent range values for the subsequent SpinBox
+                    const minMax = getContingentValues("BufferSize", "BufferSizeFieldGroup")
+
+                    // If getContingentValues() returned results, update the spin box values
+                    if (minMax[0] !== "") {
+                        rangeValuesSpinBox.from = minMax[0];
+                        rangeValuesSpinBox.to = minMax[1];
+
+                        // Explicitly set the buffer size to the rangeValuesSpinBox value in case onValueChanged doesn't trigger
+                        newFeature.attributes.replaceAttribute("BufferSize", rangeValuesSpinBox.value);
+                        saveButton.enabled = validateContingentValues();
+                    }
+                }
+            }
+
+            Text {
+                text: "Buffer Size"
+                font {
+                    bold: true
+                    pointSize: 11
+                }
+            }
+
+            Text {
+                text: rangeValuesSpinBox.from + " to " + rangeValuesSpinBox.to;
+            }
+
+            SpinBox {
+                id: rangeValuesSpinBox
+                editable: true
+                from: 0;
+                to: 0;
+                stepSize: 10
+                value: 0;
+
+                enabled: protectionComboBox.currentText !== ""
+
+                onValueChanged: {
+                    if (protectionComboBox.currentValue === "")
+                        return;
+
+                    newFeature.attributes.replaceAttribute("BufferSize", rangeValuesSpinBox.value);
+
+                    // Validate that all contingencies are valid, if so, enable the save button
+                    saveButton.enabled = validateContingentValues();
+                }
+            }
+
+            Button {
+                id: saveButton
```

Comment:
similar to the C++ sample, can this be greyed out until all input requirements are met?

Reply:
See C++ comment